Header header

std_msgs/Float32 attenuation
